apple/internal/partials/resources.bzl

```diff
@@ -109,21 +109,28 @@ def _merge_root_infoplists(
 
     return [(processor.location.content, None, depset(direct = files))]
 
-def _locales_requested(*, config_vars):
+def _locales_requested(*, locales_to_include, config_vars):
     """Determines which locales to include when resource actions.
 
     If the user has specified "apple.locales_to_include" we use those. Otherwise we don't filter.
     'Base' is included by default to any given list of locales to include.
 
     Args:
         config_vars: A dictionary (String to String) of config variables. Typically from `ctx.var`.
+        locales_to_include: A string list of locales to bundle.
 
     Returns:
         A set of locales to include or None if all should be included.
     """
-    requested_locales = config_vars.get("apple.locales_to_include")
+    config_locals_to_include = config_vars.get("apple.locales_to_include")
```

Comment:
Could you move this into the `elif` to prevent doing work if not needed?

Reply:
It'd just duplicate the call to `config_vars.get("apple.locales_to_include")` which seems less clean. The call is just to a dict, right? Should be constant time?

Reply:
Instead I would make it an `else`, and in there make this call, then use `if` on it. Even if constant, there is no need to look in the dictionary if `locales_to_include` is set.

Reply:
Sounds good